src/views/SystemIntake/RequestDetails/index.tsx

```diff
@@ -483,6 +515,146 @@ const RequestDetails = ({ systemIntake }: RequestDetailsProps) => {
           </Fieldset>
         </FieldGroup>
 
+        <FieldGroup
+          scrollElement="softwareAcquisition"
+          error={!!errors.usingSoftware}
+        >
+          <Label htmlFor="softwareAcquisition">
+            {t('requestDetails.softwareAcquisition.usingSoftwareLabel')}
+          </Label>
+          <HelpText id="elasHelpText" className="margin-top-1">
+            <Trans
+              i18nKey="intake:requestDetails.softwareAcquisition.usingSoftwareHelp"
+              components={{
+                dvsmEmail: (
+                  <UswdsLink href={`mailto:${CMS_DVSM_EMAIL}`}> </UswdsLink>
```

Comment:
for my own curiosity, does this not work?
```suggestion
                  <UswdsLink href={`mailto:${CMS_DVSM_EMAIL}`} />
```

Reply:
To my surprise it doesnt look like it does, it yells at me about no overload matching the call. Looks like it has to do with it missing the children property.